test/e2e/data/e2e_eks_ipv6_conf.yaml

```diff
@@ -0,0 +1,231 @@
+---
```

Comment:
I think we could probably use the existing EKS e2e conf file instead of creating a new one.  If we have a new conf file then we need to invoke the e2e tests separately.....which sometimes needs to be done for experimental features.

Reply:
Let's talk about it when you get back. :)

Reply:
Discussed today